### PR TITLE
[Oxfordshire] Add asset layers

### PR DESCRIPTION
**Pull request by struan**
Add some asset layers to the OCC site. At the moment only include the asset data in the open311 description or email, pending back end integration.

[skip changelog]

### CODE REVIEW

File: web/cobrands/oxfordshire/assets.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ var defaults = {
     },
     asset_type: 'spot',
     max_resolution: 4.777314267158508,
-    min_resolution: 0.5971642833948135,
+    min_resolution: 0.00001,
```
Comment (davea) [inline web/cobrands/oxfordshire/assets.js]: That `fix-assets-disappearing-at-max-zoom` branch has been merged now :)

File: web/cobrands/oxfordshire/assets.js [hunk 2]
```diff
@@ -50,4 +50,30 @@ fixmystreet.assets.add(defaults, {
...
 
+fixmystreet.assets.add(defaults, {
+    // have to do this by hand rather than using wfs_* options
+    // as the server does not like being POSTed xml with application/xml
+    // as the Content-Type which is what using those options results in.
+    http_options: {
+        headers: {
+            'Content-Type': 'text/plain'
+        },
+        url: "https://tilma.staging.mysociety.org/proxy/occ/wfs",
```
Comment (davea) [inline web/cobrands/oxfordshire/assets.js]: Should this hostname be dependent on live/staging like `tilma_url` above?

File: templates/email/oxfordshire/submit.txt [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+[% closest_address %]----------
+
+Replies to this email will go to the user who submitted the problem.
+
+[% signature %]
+
+----------
+
+Never retype another FixMyStreet report
```
Comment (davea) [inline templates/email/oxfordshire/submit.txt]: Think from here down can be removed, as it's a council who have an integration.

**Comment by struan** [comment]
I was going to squash it down before merging

**Pull request closed without merging**